### DISCUSSION TITLE
Second press of CMD+A should select parent list after instead of selecting whole tree

### DISCUSSION THREAD

**Discussion by vslinko** [vslinko/obsidian-outliner#58] [Ideas]

**Comment by ShreyasKallingal** [comment]
On second thought, I think this feature would be unnecessarily complicated since child nodes don't usually have information about their parents. It seems that other outlining apps work as your plugin does now. I think that is more than enough—thank you very much!

**Reply by laphylee** [reply]
@ShreyasKallingal What do you mean by "nodes don't usually have information about their parents"?

@vslinko  Actually I like this idea. At the moment, **it's hard to select the current list.**  

I tried CMD+A on Dynalist, its selection queue is more advanced:

1. current line
2. current list (select children)
3. current block (without parent)
4. current block (with parent)
5. up-level block

![GIF](https://user-images.githubusercontent.com/13502835/152119898-a309a158-55f8-4e16-b35e-64cd06633c81.gif)

**Reply by vslinko** [reply]
I think I can implement behavior like in Dynalist. It's not much harder than the original idea.

**Reply by filyp** [reply]
This would be awesome!

What would be even more useful for me, would be some command to select current list, that I can bind to some key myself.
